TESTING.md

```diff
@@ -232,9 +232,18 @@ Sets the number of iterations each test uses by default.
 
 ### Running tests in the browser
 
-Normally we use `CLIENT=firefox` to run a set of tests in the browser
-automatically. This opens a browser window, automatically runs the requested
-tests in it, and reports the results back to the shell.
+To run tests in the browser, you first have to install playwright:
+
+```shell
+npx playwright install
+```
+
+This will download the `firefox`, `chromium` and `webkit` `CLIENT`s onto
+your system.
+
+PouchDB is tested with `CLIENT=firefox`, `CLIENT=chromium` and `CLIENT=webkit`
+to run a set of tests in the browser automatically. This runs these browsers
+in a “headless” mode and prints the test results back into the terminal.
```

Comment:
Does this paragraph contain enough info for someone unfamiliar with the project?  Would including the test command in the examples be more helpful?  E.g. `CLIENT=firefox npm test`

Reply:
that’s fair! will fix